index.html

```diff
@@ -241,5 +241,35 @@ <h2>Examples</h2>
         ...
       </pre>
     </section>
+    <section class="informative">
+      <h2>Privacy and Security Considerations</h2>
```

Comment:
The W3C seems to ask two *separate* sections for security and privacy. I know others spec don't do that and it looks a bit silly to do it especially for this spec where we don't have much to say. But I also know [W3C explicitly asked about for a recent spec](https://github.com/w3c/mathml-core/issues/48)... I'm not exactly sure there is always a clear separation between privacy/security , but checking the questionnaire there are questions about exposing information (what we do here) that looks related to privacy and others that seem more related to security (where we basically reply "not a problem"). So I was thinking put everything in privacy section and "no known issue" in security section...

Reply:
We can split these if we're asked later. For the time being, my humble opinion is that this is good enough as is. Some specs don't even have such a section at all ([example](https://w3c.github.io/mediacapture-handle/identity/index.html)), so it seems like a matter of polish rather than a strict requirement.

Reply:
OK.